CMakeLists.txt

```diff
@@ -1,17 +1,13 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Add comment about WHY we're picking CMake 3.9. Because it's the latest supported version in  ... RHEL5? Amazon Linux 1?